training/training/routes/image/image.py

```diff
@@ -0,0 +1,58 @@
+from typing import Literal, Optional
+from django.http import HttpRequest
+from ninja import Router, Schema
+from training.core.criterion import getCriterionHandler
+from training.core.dl_model import DLModel
+from training.core.dataset import ImageDefaultDatasetCreator
+from torch.utils.data import DataLoader
+from training.core.optimizer import getOptimizer
+from training.core.trainer import ClassificationTrainer
+from training.routes.image.schemas import ImageParams
+from training.core.authenticator import FirebaseAuth
+
+router = Router()
+
+
+@router.post("", auth=FirebaseAuth())
+def imageTrain(request: HttpRequest, imageParams: ImageParams):
+    if imageParams.default:
+        dataCreator = ImageDefaultDatasetCreator.fromDefault(
+            imageParams.default
+        )
+        print(vars(dataCreator))
+        train_loader = dataCreator.createTrainDataset()
+        test_loader = dataCreator.createTestDataset()
+        # train_loader = DataLoader(
+        #     dataCreator.createTrainDataset(),
+        #     batch_size=imageParams.batch_size,
+        #     shuffle=False,
+        #     drop_last=True,
+        # )
+
+        # test_loader = DataLoader(
+        #     dataCreator.createTestDataset(),
+        #     batch_size=imageParams.batch_size,
+        #     shuffle=False,
+        #     drop_last=True,
+        # )
```

Comment:
is this not neeed?

Reply:
No. Line 23 and 24 already take care of it. Just removed it.